tests/components/switchbot/test_config_flow.py

```diff
@@ -22,7 +32,7 @@ async def test_user_form_valid_mac(hass):
     result = await hass.config_entries.flow.async_init(
         DOMAIN, context={"source": SOURCE_USER}
     )
-    assert result["type"] == FlowResultType.FORM
+    assert result["type"] == RESULT_TYPE_FORM
```

Comment:
Please revert this change. You may need to rebase on latest dev branch. `FlowResultType` is the new way and `RESULT_TYPE_FORM` is legacy.